src/main/java/org/example/daos/AmazonS3Connector.java

```diff
@@ -0,0 +1,57 @@
+package org.example.daos;
+
+import com.amazonaws.ClientConfiguration;
+import com.amazonaws.Protocol;
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.auth.DefaultAWSCredentialsProviderChain;
+import com.amazonaws.client.builder.AwsClientBuilder;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+
+public final class AmazonS3Connector {
+
+    private static final String REGION = "eu-west-1";
+    private static AmazonS3 amazonS3Client;
+    private static final String ENDPOINT_URL = System.getenv("ENDPOINT_URL");
+
+    private AmazonS3Connector() {
+    }
```

Comment:
it's default constructor -> do we need to define it? It should be autoprovided underneath by Java

Reply:
It is a util class. It needs this constructor to adhere to linter requirements.
Added 
throw new UnsupportedOperationException("This is a utility class and cannot be instantiated");
in this constructor as it was done in JwtUtils class' constructor.